InterfaceParser/parsers/parse_error.py

```diff
@@ -0,0 +1,11 @@
+"""Parse error.
+
+"""
+
+
+class ParseError(Exception):
+    """Parse error.
+
+    This exception is raised when XML contains errors and can't be parsed.
```

Comment:
Should there be some implementation in this class?

Reply:
This error used for exceptional termination of parser.It appear only in case when the incorrect/corrupted *.xml provided to parser. In such case parser won't be able to extract correct model, so it operation will be interrupted with exception. In all other cases this exception will not be raised. Do you have any idea how should we handling such exception?